ComputeSharp.sln

```diff
@@ -48,6 +48,8 @@ Project("{6EC3EE1D-3C4E-46DD-8F32-0CC8E7565705}") = "ComputeSharp.Sample.FSharp"
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ComputeSharp.Sample.FSharp.Shaders", "samples\ComputeSharp.Sample.FSharp.Shaders\ComputeSharp.Sample.FSharp.Shaders.csproj", "{1F37A704-3CDF-45BB-8114-ED9137E5F2CF}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ComputeSharp.ShaderToImage", "samples\ComputeSharp.ShaderToImage\ComputeSharp.ShaderToImage.csproj", "{F0FA942A-076E-43C1-B84B-0294FA8DCD91}"
```

Comment:
I wish you had asked me about doing this before doing so 😅
I don't want a project just to generate assets for testing.
Could you please rebase and drop this commit?

Reply:
I mean I used it for testing, but it's still a practical sample based off of the example in the [readme here](https://github.com/Sergio0694/ComputeSharp#working-with-images). We could have the similar prompt like the SwapChain sample, but figured it'd just be simpler to show it working with a single one for now.

Reply:
It's used for testing but it's not a project that actually gets used. It's not an actual test to run in the CI nor a sample to use, it's just a one-liner that you need to run to create the .png while creating a new sample. I don't feel like it's worth adding to the solution, especially not now that I have a bazillion branches open and I'll already have to deal with tons of merge conflicts in the solution file when trying to put everything back together 🙂

Reply:
Since it's just three lines will add to readme instead.